app/src/sharedTest/java/org/oppia/app/profileprogress/ProfileProgressFragmentTest.kt

```diff
@@ -281,8 +362,25 @@ class ProfileProgressFragmentTest {
 
   @Test
   fun testProfileProgressActivity_recyclerViewItem1_chapterNameIsCorrect() {
-    launch<ProfileProgressActivity>(createProfileProgressActivityIntent(0)).use {
+    launch<ProfileProgressActivity>(createProfileProgressActivityIntent(internalProfileId)).use {
       onView(withId(R.id.profile_progress_list)).perform(scrollToPosition<RecyclerView.ViewHolder>(1))
+      waitForTheView(withText("What is a Fraction?"))
+      onView(atPositionOnView(R.id.profile_progress_list, 1, R.id.chapter_name_text_view)).check(
+        matches(
+          withText(
+            containsString("What is a Fraction?")
+          )
+        )
+      )
```

Comment:
`matches` method can be written in single line. Please make same correction at other places in file.

Reply:
Indentation in such cases is something which depends on developer's choice as everything is correct unless and until it crosses 80 character mark in a single line.
The main point should be that the indentation in a single file should be consistent.